### PR TITLE
EMSUSD-1504 prevent warnings when reloading the plugin

### PR DESCRIPTION
**Pull request by pierrebai-adsk**
Use our own version of the function to register string resources to avoid spamming warning about duplicate resources.

### CODE REVIEW

**Review by AramAzhari-adsk — DISMISSED** [review]

**Review by seando-adsk — CHANGES_REQUESTED** [review]
How do the existing MEL files that call `getMayaUsdLibString` work now? You've deleted the MEL file `mayaUsdLibRegisterStrings.mel` that contained this proc. I think those scripts will error now.

**Comment by pierrebai-adsk** [comment]
Yes, that is a mistake. I must not have opened a window that depended on them when testing. I fill revert that part.

**Comment by pierrebai-adsk** [comment]
The reason I missed it is that they are only used in a single dialog: Add Maya Reference. Added them back and tested that window. Now it works again.

**Review by AramAzhari-adsk — APPROVED** [review]

**Review by seando-adsk — APPROVED** [review]

File: test/lib/mayaUsd/nodes/testProxyShapeConversion.py [hunk 1]
```diff
@@ -50,9 +50,11 @@ def setUp(self):
         """Clear the scene and setup the Maya preferences."""
         cmds.file(f=True, new=True)
         # Make sure up-axis is Z.
-        cmds.upAxis(axis='z')
+        if cmds.upAxis(query=True, axis=True).lower() != 'z':
```
Comment (AramAzhari-adsk) [inline test/lib/mayaUsd/nodes/testProxyShapeConversion.py:53]: Do these occur when reloading the plugin?

**Review by AramAzhari-adsk — APPROVED** [review]

File: test/lib/mayaUsd/nodes/testProxyShapeConversion.py (hunk 1 above)
Comment (pierrebai-adsk) [reply test/lib/mayaUsd/nodes/testProxyShapeConversion.py:53]: The commands were issuing warnings that the up-axis was already Z, I'm just reducing the amount of noise when running the tests.

**Pull request merged**